tests/e2e/tslint.json

```diff
@@ -52,26 +52,15 @@
       "trace"
     ],
     "no-construct": true,
-    "no-parameter-properties": false,
     "no-debugger": true,
     "no-duplicate-variable": true,
     "no-empty": true,
     "no-eval": true,
     "no-string-literal": true,
     "no-switch-case-fall-through": true,
-    "trailing-comma": [true, {
```

Comment:
Is there similar rule available in TsLint 6.x?

Reply:
oh, looks like we miss it in downstream test automation project. I will back it here and will add there too